draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2207,7 +2220,7 @@ uCDN-A                         dCDN-B                         dCDN-C
         <vspace blankLines="1"/>
         Value: UTCWindow object as defined in <xref target="utc-window"/>.
         <vspace blankLines="1"/>
-        Mandatory: No, but exactly one of "unixEpochWindow" or "utcWindow" MUST be present.
+        Mandatory: No, but exactly one of the "unixEpochWindow" or "utcWindow" MUST be present.
```

Comment:
Adding "the" sounds more awkward here.  We're referring to one of these choices.  Maybe "either" is is a better choice than "the".